starter-code/src/viking.js

```diff
@@ -1,11 +1,88 @@
 // Soldier
-function Soldier () {}
-
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+Soldier.prototype.attack = function (){
+  return  this.strength ;
+};
+Soldier.prototype.receiveDamage = function (damage){
+  this.health = this.health - damage;
```

Comment:
Esta operación está perfecta, pero también puedes hacer algo así:
this.health -= damage;
Y escribes menos de código :)